com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -196,6 +198,8 @@ public Color clearColor
         ClearFlag m_ClearFlag = ClearFlag.None;
         Color m_ClearColor = Color.black;
 
+        public DebugHandler DebugHandler { get; set; }
```

Comment:
does it need to be public?

Reply:
I also see the renderer has a reference to debug handled. Depending on where you access this you have access to the renderer in `RenderingData` struct.

Reply:
I made it internal in my branch.

I think you're right, based on a quick look, all use cases of this property also have access to `RenderingData` and thus the debugHandler via `renderingData.cameraData.renderer.DebugHandler`. I'll add that to be done in a follow-up PR.

Reply:
It's now internal, suggested refactor will be made later (https://jira.unity3d.com/browse/XPIPELINE-157).